autogen/docs/behaviorspace.md.mustache

```diff
@@ -296,17 +296,20 @@ the "Run" button. A dialog titled "Run options" will appear.
 
 #### Run options: formats
 
-The run options dialog lets you type or **Browse...** for the files for the formats you would
-like the data from your experiment saved in (if any).  If you don't wish to use a particlar
-format, you can clear the file text or click the **Disable** button and it'll be cleared for
-you. Data is collected for each run or step, according to the setting of **Measure runs at
-every step** option.  For both formats the initial state of the system is recorded, after the
-**setup** commands run but before the **go** commands run for the first time.
-
-Both **Table output** and **Spreadsheet output** will include a header section that has rows for
-1) the NetLogo app version used, 2) the name of the NetLogo model file used, 3) the name of the
-BehaviorSpace experiment used, 4) the date and time at the start, and 5) the dimensions of the
-world used at the start.
+The run options dialog lets you choose to create data output files in two formats, **Table
+output** and **Spreadsheet output**. If one or both of these formats is selected, you can
+also select the supplementary **Lists output**. Each file path can be entered in its
+corresponding text box, or using the **Browse...* button to select a file path using the system
+file dialog. If you don't wish to use a particular format, you can clear the file text or click
+the **Disable** button and it'll be cleared for you. Data is collected for each run or step,
+according to the setting of **Measure runs at every step** option. For **Table output** and
```

Comment:
Adjust for new wording and reporter option for "Run metrics", which replaces **Measure runs at every step**